nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+
+const app = express();
+
+app.get("/meals", (req, res)=>{
+    res.json(meals);
```

Comment:
The /meals route needs to return a collection of meals that need to be expanded with reviews. If a meal doesn't have any reviews one should have an empty array:

meals.reservations = [];

if a meal has reviews you should add them to that property:

meals.reservations = [{
  //review...
}]

these reviews have to be obtained from the /data/reviews.json file and matched by id:

For a review be contained within a meal meal.id === review.mealId